### PR TITLE
RHTAPBUGS-362: Upgrade e2e, infra-deployments to 4.12.9 and change workflow

### PR DESCRIPTION
**Pull request by flacatus**

### CODE REVIEW

**Comment by flacatus** [comment]
/pj-rehearse pull-ci-redhat-appstudio-infra-deployments-main-appstudio-hac-e2e-tests

**Comment by flacatus** [comment]
/pj-rehearse pull-ci-redhat-appstudio-infra-deployments-main-appstudio-e2e-tests

**Comment by flacatus** [comment]
/pj-rehearse pull-ci-redhat-appstudio-infra-deployments-main-appstudio-hac-e2e-tests

**Comment by flacatus** [comment]
/pj-rehearse pull-ci-redhat-appstudio-e2e-tests-main-load-test-ci-100u-1t

**Comment by flacatus** [comment]
/pj-rehearse pull-ci-redhat-appstudio-infra-deployments-main-appstudio-hac-e2e-tests

Fail not related with vcluster

**Comment by flacatus** [comment]
/pj-rehearse pull-ci-redhat-appstudio-infra-deployments-main-appstudio-e2e-tests

**Comment by flacatus** [comment]
/pj-rehearse pull-ci-redhat-appstudio-infra-deployments-main-appstudio-e2e-tests

**Comment by flacatus** [comment]
/pj-rehearse pull-ci-redhat-appstudio-infra-deployments-main-appstudio-hac-e2e-tests

**Comment by flacatus** [comment]
/pj-rehearse pull-ci-redhat-appstudio-infra-deployments-main-appstudio-hac-e2e-tests

**Comment by flacatus** [comment]
/pj-rehearse pull-ci-redhat-appstudio-infra-deployments-main-appstudio-e2e-tests

**Comment by flacatus** [comment]
/pj-rehearse pull-ci-redhat-appstudio-e2e-tests-main-load-test-ci-100u-1t

**Comment by psturc** [comment]
/pj-rehearse pull-ci-redhat-appstudio-e2e-tests-main-redhat-appstudio-e2e

**Comment by flacatus** [comment]
/pj-rehearse pull-ci-redhat-appstudio-e2e-tests-main-redhat-appstudio-e2e

**Comment by flacatus** [comment]
/pj-rehearse pull-ci-redhat-appstudio-infra-deployments-main-appstudio-e2e-tests

**Comment by flacatus** [comment]
/pj-rehearse pull-ci-redhat-appstudio-infra-deployments-main-appstudio-e2e-tests

**Comment by flacatus** [comment]
/pj-rehearse pull-ci-redhat-appstudio-e2e-tests-main-redhat-appstudio-e2e

**Comment by flacatus** [comment]
/pj-rehearse pull-ci-redhat-appstudio-e2e-tests-main-redhat-appstudio-e2e

File: ci-operator/config/redhat-appstudio/e2e-tests/redhat-appstudio-e2e-tests-main.yaml [hunk 1]
```diff
@@ -45,9 +45,10 @@ tests:
 - as: redhat-appstudio-e2e
   skip_if_only_changed: ^docs/|^\.github/|^\.tekton/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
   steps:
-    cluster_profile: aws
+    cluster_profile: aws-2
     env:
-      HYPERSHIFT_HC_RELEASE_IMAGE: quay.io/openshift-release-dev/ocp-release:4.12.4-multi
+      HYPERSHIFT_HC_RELEASE_IMAGE: quay.io/openshift-release-dev/ocp-release:4.12.9-multi
```
Comment (psturc) [inline ci-operator/config/redhat-appstudio/e2e-tests/redhat-appstudio-e2e-tests-main.yaml:50]: as we're updating it, should we go to the latest patch release?
https://amd64.ocp.releases.ci.openshift.org/releasestream/4-stable/release/4.12.19

Comment (psturc) [reply ci-operator/config/redhat-appstudio/e2e-tests/redhat-appstudio-e2e-tests-main.yaml:50]:
```suggestion
      HYPERSHIFT_HC_RELEASE_IMAGE: quay.io/openshift-release-dev/ocp-release:4.12.19-multi
```

**Comment by psturc** [comment]
/pj-rehearse pull-ci-redhat-appstudio-infra-deployments-main-appstudio-hac-e2e-tests

**Comment by flacatus** [comment]
https://issues.redhat.com/browse/RHTAPBUGS-362

**Pull request merged**